kits/movie-vocab-buddy/prompts/weekly-quiz_instructor-llmnode-647_user_1.md

```
@@ -0,0 +1 @@
+Woords saved over the past few weeks: {{postgresNode_140.output.queryResult}}
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)